cppapi/server/fwdattribute.cpp

```diff
@@ -133,10 +133,10 @@ void FwdAttribute::set_att_config(const Tango::AttributeConfig_5 &conf)
 // Enum labels (in case of)
 //
 
-    enum_labels.clear();
```

Comment:
This new indentation looks strange on github page showing the differences (this page).
It was not aligned before, but it still does not seems to be aligned with the code above (line 130) after.

Reply:
I guess this is again due to the mixing between tabs and spaces.